app/helpers/request_helper.py

```diff
@@ -102,3 +102,13 @@ def get_doc_from_store(tx_id):
     if response_ok(response):
         logger.info("Successfully got document from store")
         return response.json()
+
+
+def get_feedback_from_store(feedback_id):
+    logger.info("About to get feedback from store")
```

Comment:
The id should be added to this:
logger.info("About to get feedback from store", id=id)